abi_wrapper.go

```diff
@@ -0,0 +1,605 @@
+package web3
+
+import (
+	"errors"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gochain-io/gochain/v3/accounts/abi"
+)
+
+func ABIBuiltIn(contractFile string) (io.Reader, error) {
+	if val, ok := bundledContracts[contractFile]; ok {
+		return strings.NewReader(val), nil
+	}
+	return nil, errors.New("Cannot find bundled contract")
```

Comment:
What about just returning `nil` for not found? Especially if this is expanded to parse the json as well - then nil can be checked easily, instead of inspecting the error.